qutip/core/coefficient.py

```diff
@@ -245,6 +245,9 @@ class CompilationOptions(QutipOptions):
 
     clean_on_error : bool [True]
         When writing a cython file that cannot be imported, erase it.
+
+    build_dir: str [None]
+        cythonize's build_dir.
```

Comment:
Might be worthwhile mentioning Cython's default and why one might need to set this (e.g. that the files there are cached).

Reply:
While the parameter exist in the source code, it is not in the [documentation](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#cythonize-arguments) (yet?)
I don't want to give more details than cython themselves give.